.github/workflows/application-test.yaml

```diff
@@ -0,0 +1,41 @@
+name: Application Specific Test
+on:
+  pull_request:
+    types: [synchronize, labeled]
+
+jobs:
+  setup:
+    name: Run Application Install Tests
+    runs-on:
+      - ubuntu-latest
+    if: contains(join(github.event.pull_request.labels.*.name, ' '), 'services/')
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version-file: 'apptests/go.mod'
+
+      - name: Install Ginkgo
+        working-directory: apptests
+        run: go install github.com/onsi/ginkgo/v2/ginkgo
+
+      - name: Run Ginkgo Install Tests
+        working-directory: apptests
+        run: |
+          # Extract labels from the pull request event payload
+          PR_LABELS=$(echo "${{ join(github.event.pull_request.labels.*.name, ' ') }}")
+          echo $PR_LABELS
+          KAPP_NAME=""
+          for label in $PR_LABELS; do
+            echo $label
+            if [[ "$label" == "services/"* ]]; then
+              KAPP_NAME=$(echo "$label" | cut -d '/' -f 2)
+              break
+            fi
+          done
+
+          ginkgo --label-filter="install && $KAPP_NAME" appscenarios
```

Comment:
I'm wondering why not just not running it on services that have changes, wdyt?

Reply:
We have [this ticket](https://d2iq.atlassian.net/browse/D2IQ-100345) for running on the changes. Technically, if we have that, we don't need the labels to trigger specific individual tests explicitly.

I am thinking we need three ways to run the tests:
- An auto-triggered action that runs on PRs - this will run tests apps that have changed.
- A label/action that will trigger a specific test. (This would be helpful to override the change detection and run or re-run other tests.)
- A label/actions that will run all suites/tests.
Technically, the first two have some overlapping functionality.

wdyt?

Reply:
I think relying on label is an easy way out since bump PRs like https://github.com/mesosphere/kommander-applications/pull/1964 is automatically labeled. 

vs. 

PR like this one(no change on k-apps but only GHA) will not be automatically labeled -- you can see the history I manually labeled this PR.

Reply:
I think this works well. We already have the change detection and just use the labels to run the actions.

Reply:
addressed to refactor to trigger GHA via matrix of (app, install || upgrade).

Also added workflow dispatch on both the parent job to trigger for ALL apps, 

as well as to the individual scenario GHA of a single app on a single action.